server-side/login.test.js

```diff
@@ -0,0 +1,68 @@
+import request from 'supertest';
+import app from '../app.js'; 
+import mongoose from 'mongoose';
+import Users from '../models/user.model.js';
+import bcrypt from 'bcrypt';
+
+import { connectMongo, disconnectMongo } from '../config/db.js';
```

Comment:
you shouldn't connect to the real DB for testing, for tests you should create mock DB and use him